docs/get-started/viam.md

```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 images: ["/general/understand.png"]
 ---
 
-Viam is a complete software platform that supports every step of your {{< glossary_tooltip term_id="smart-machine" text="smart machine">}} development lifecycle.
+Viam is a software platform that makes it easy to work with hardware and software.
+At the core of Viam is the open-source `viam-server` executable which:
 
-## Plan your machine
+- Creates, configures, and maintains any machine on which it is installed
+- Securely handles all communications.
+- Runs drivers, custom code, and any other software.
+- Accepts API requests.
+- Runs {{< glossary_tooltip term_id="service" text="services">}} like computer vision, motion planning, data management, machine learning, and more.
 
-When using Viam, this is what you'll need to know to plan your machine:
+`viam-server` **runs on Linux and macOS** and supports a wide variety of popular systems, including:
 
-<img src="https://assets-global.website-files.com/62fba5686b6d47fe2a1ed2a6/633d91b848050946efcf0690_viam-overview-illustrations-build.svg" alt="A diagram of machine parts and software" class="alignright" style="width:250px;"></img>
+{{< board-carousel >}}
+<br>
 
```

Comment:
Ahh cool I think the concrete examples will help it click for people. I think we can fit just a smidge more broad-strokes explanation of what Viam is first, and then also include some sort of transition to the examples like "Here are some examples to show how you can use Viam in your projects..."

I like that people won't have to scroll too far before they see concrete things, but I think we just need a little more first.